### ISSUE TITLE
When the slider items are focusable, there should be an easy way for the keyboard user to skip tabbing through each focusable item, to move to the next focusable element on the page

### ISSUE THREAD

**Issue by gaurav5430** [Accessible360/accessible-slick#55]
If there is a slider with 10 items, similar to the product items slider in codepen examples: https://codepen.io/A360/pen/mdPBdEz
then by default all the items would be part of the tab navigation flow, so when the user tabs into the slider, they will tab into the nav buttons, and then on to each individual slide. This might be ok for smaller lists, but if it is a bigger list, the user would be frustrated as they need to tab through each item to finally skip the slider and go to the next focusable thing on the page. 

Is that something that has been considered? Is it possible to do that in the current setup ?

(This is generally solved by roving tab index, or aria-activedescendent in some cases)

**Comment by mikemccaffrey** [comment]
Only the slides that are currently visible can be tab focused. In the linked example, only four of the slides can be focused, and you need to use the next button to get to the rest of the slides. Do you really have a slideshow with 10 items visible at once?
